draft-ietf-acme-acme.md

```diff
@@ -826,19 +826,13 @@ applications until the authorization expires. [[ Open issue: More flexible
 scoping? ]]
 
 challenges (required, array):
-: The challenges that the client needs to fulfill
+: The challenges that the client may to fulfill
```

Comment:
This doesn't parse as-is.  Suggest "The challenges that the client can fulfill", with clarification that any one of them is sufficient below.